apps/CA.pl.in

```diff
@@ -77,6 +91,19 @@ sub copy_pemfile
     return $found == 2 ? 0 : 1;
 }
 
+sub copy_any
```

Comment:
It obviously shortens the codes. But I wonder if it is reasonable to load File::Copy module for a few utilization...